cms/static/js/views/utils/create_course_utils.js

```diff
@@ -11,6 +11,14 @@ define(["jquery", "gettext", "common/js/components/utils/view_utils", "js/views/
 
             CreateUtilsFactory.call(this, selectors, classes, keyLengthViolationMessage, keyFieldSelectors, nonEmptyCheckFieldSelectors);
 
+            this.setupOrgAutocomplete = function(){
+                $.getJSON('/org_names_list', function (data) {
```

Comment:
I would prefer we model this endpoint as a resource -- is it possible to do "/organizations" here?